packages/webapp/src/components/Finances/Transaction/ExpandedContent/styles.module.scss

```diff
@@ -0,0 +1,40 @@
+/*
+ *  Copyright 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+
+// ExpandedContent
+.expandedContent {
+  margin: 0 5px 6px 5px;
```

Comment:
Even though this values match the Figma, I'd recommend rounding them to `0 4px 8px 4px` or something similar. Loïc has mentioned a couple of times that he's trying to keep things in an 8px grid.